File: php/secure-public-access/GetArticle.php
<?php
// DocSection: securing_public_access_get_article
// Tip: Find more about PHP SDKs at https://kontent.ai/learn/php
// Defined by Composer to include required libraries
require __DIR__ . '/vendor/autoload.php';

use Kontent\Ai\Delivery\DeliveryClient;

$client = new DeliveryClient('<YOUR_PROJECT_ID>', null, '<YOUR_API_KEY>');

$item = $client->getItem('my_article');
// EndDocSection
?>

File: java/secure-public-access/GetArticle.java
// DocSection: securing_public_access_get_article
// Tip: Find more about Java SDK at https://kontent.ai/learn/java
import kontent.ai.delivery.*;

// Initializes a DeliveryClient with secure access key
DeliveryClient client = new DeliveryClient(
	DeliveryOptions.builder()
		.projectId("<YOUR_ENVIRONMENT_ID>")
		.productionApiKey("<YOUR_API_KEY>")
		.build()
);

// Gets the latest version of an item
CompletionStage<ContentItemResponse> item = client.getItem("my_article");
// EndDocSection